src/core/cgn.tf

```diff
@@ -122,3 +122,74 @@ module "cgn_cosmos_db" {
   account_name        = module.cosmos_cgn.name
 }
 
+
+## Blob storage due to legal backup
+#tfsec:ignore:azure-storage-default-action-deny
+module "cgn_legalbackup_storage" {
+  source = "git::https://github.com/pagopa/azurerm.git//storage_account?ref=v1.0.79"
+
+  name                       = replace(format("%s_legalbackup_cgn_storage", local.project), "-", "")
```

Comment:
```suggestion
  name                       = replace(format("%s-legalbackup-cgn-storage", local.project), "-", "")
```

Reply:
@pasqualedevita renamed all name and resources by https://github.com/pagopa/io-infra/pull/84/commits/2a5244310bea68b850590847b92065c6a48ad99e